content/accounts.md

```diff
@@ -0,0 +1,700 @@
+
+1. [`Accounts.resetPassword`](http://docs.meteor.com/#/full/accounts_resetpassword) - this one should be used both for resetting the password, and enrolling a new user; it accepts both kinds of tokens.
+2. [`Accounts.verifyEmail`](http://docs.meteor.com/#/full/accounts_verifyemail)
+
+After you are have called one of the two functions above or the user has cancelled the process, call the `done` function you got in the link callback. This will tell Meteor to get out of the special state it enters when you're doing one of the email account flows.
+
+<h3 id="customizing-emails">Customizing accounts emails</h3>
+
+You will probably want to customize the emails `accounts-password` will send on your behalf. This can be easily done through the [`Accounts.emailTemplates` API](http://docs.meteor.com/#/full/accounts_emailtemplates). Below is some example code from the Todos app:
+
+```js
+Accounts.emailTemplates.siteName = "Meteor Guide Todos Example";
+Accounts.emailTemplates.from = "Meteor Todos Accounts <accounts@example.com>";
+
+Accounts.emailTemplates.resetPassword = {
+  subject(user) {
+    return "Reset your password on Meteor Todos";
+  },
+  text(user, url) {
+    return `Hello!
+Click the link below to reset your password on Meteor Todos.
+${url}
+If you didn't request this email, please ignore it.
+Thanks,
+The Meteor Todos team
+`
+  },
+  html(user, url) {
+    // This is where HTML email content would go.
+    // See the section about html emails below.
+  }
+};
+```
+
+As you can see, we can use the ES2015 template string functionality to generate a multi-line string that includes the password reset URL. We can also set a custom `from` address and email subject.
+
+<h4 id="html-emails">HTML emails</h4>
+
+If you've ever needed to deal with sending pretty HTML emails from an app, you know that it can quickly become a nightmare. Compatibility of popular email clients with basic HTML features like CSS is notoriously spotty, so it is hard to author something that works at all. Start with a [responsive email template](https://github.com/leemunroe/responsive-html-email-template) or [framework](http://foundation.zurb.com/emails/email-templates.html), and then use a tool to convert your email content into something that is compatible with all email clients. [This blog post by Mailgun covers some of the main issues with HTML email.](http://blog.mailgun.com/transactional-html-email-templates/)
+
+<h2 id="oauth">OAuth login</h2>
+
+In the distant past, it could have been a huge headache to get Facebook or Google login to work with your app. Thankfully, most popular login providers have standardized around some version of [OAuth](https://en.wikipedia.org/wiki/OAuth), and Meteor supports some of the most popular login services out of the box.
```

Comment:
Haha, it's a bit of a joke :P I actually remember this being a hassle when I was building my first rails app.